src/ansys/simai/core/data/projects.py

```diff
@@ -255,3 +265,26 @@ def delete(self, project: Identifiable[Project]) -> None:
             project: ID or :class:`model <Project>` of the project.
         """
         self._client._api.delete_project(get_id_from_identifiable(project))
+
+    def cancel_build(self, project: Identifiable[Project]):
```

Comment:
Or is it build ?
We need to pick. (To be transparent, I'm confused as well on which one we're supposed to use)

Reply:
- the endpoint is `projects/{project_id}/cancel-training`
- the UI refers to it as `Cancel Build` universally, ticket refers to it as `cancel build` and reference to `models.build` etc